api_tests/users/views/test_user_external_identities.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from osf_tests.factories import AuthUserFactory
+from api.base.settings.defaults import API_BASE
+
+
+@pytest.fixture()
+def user():
+    user = AuthUserFactory()
+    user.external_identity = {
+        'ORCID': {
+            '0000-0001-9143-4653': 'VERIFIED'
+        },
+        'LOTUS': {
+            '0000-0001-9143-4652': 'LINK'
+        }
+    }
+    user.save()
+    return user
+
+
+@pytest.mark.django_db
+class TestUserIdentitiesList:
+
+    @pytest.fixture()
+    def url(self, user):
+        return '/{}users/{}/settings/identities/'.format(API_BASE, user._id)
+
+    def test_authorized_gets_200(self, app, user, url):
```

Comment:
Expected response format is incorrect (doesn't adhere to JSONAPI spec).

Recommended response: id is `external_id_provider`, type is `external_identities`, and then you have two attributes that should be nested under attributes.  The self link refers to itself for each item in the list, which is the detail link, not the list link.
```
{
    "data": [
        {
            "links": {
                "self": "http://localhost:8000/v2/users/96ska/settings/identities/LOTUS/"
            },
            "attributes": {
                "status": "LINK",
                "external_id": "0000-0001-9143-4652"
            },
            "type": "external_identities",
            "id": "LOTUS"
        },
        {
            "links": {
                "self": "http://localhost:8000/v2/users/96ska/settings/identities/ORCID/"
            },
            "attributes": {
                "status": "VERIFIED",
                "external_id": "0000-0001-9143-4653"
            },
            "type": "external_identities",
            "id": "ORCID"
        }
    ],
    "meta": {
        "total": 2,
        "per_page": 10,
        "version": "2.8"
    },
    "links": {
        "self": "http://localhost:8000/v2/users/96ska/settings/identities/",
        "first": null,
        "last": null,
        "prev": null,
        "next": null
    }
}
```